blazor/chart/chart-print.md

```diff
@@ -373,6 +373,134 @@ If you set `IsBase64` parameter to **true** while calling the [ExportAsync](http
 
 ```
 
+### BeforeExport ExcelProperties
```

Comment:
Event name is `Exporting` but you have used the title as BeforeExport. The title is not proper. Use the below title.

**Customizing the exported Chart using Exporting event**